skbio/io/format/stockholm.py

```diff
@@ -577,3 +576,79 @@ def _check_for_malformed_line(line, expected_len):
         raise StockholmFormatError('Line contains %d item(s). It must '
                                    'contain exactly %d item(s).'
                                    % (len(line), expected_len))
+
+
+@stockholm.writer(TabularMSA)
+def _tabular_msa_to_stockholm(obj, fh):
+    index = list(obj.index)
+    # Writes header
+    fh.write("# STOCKHOLM 1.0\n")
+
+    # Writes GF data to file
+    if obj.has_metadata():
+        for i in range(len(obj.metadata)):
+            gf_feature = list(obj.metadata.keys())[i].strip("'")
+            if gf_feature == 'NH' and (type(obj.metadata[gf_feature]) is
```

Comment:
This type-check will only work for `OrderedDict`. While the Stockholm reader uses `OrderedDict`, we still want this to work with any kind of dict in case the user created this structure on their own and didn't use `OrderedDict`:

```python
if feature_name == 'NH' and isinstance(feature_data, dict):
    ...
```